spec/lib/radar_base_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe Radar::Base do
+  let(:radar) {
+    Radar::Base.new(RadarSetting.create())
+  }
+
+  # Test case for GH #591
+  it "should be do proper type casts" do
+    expect(radar.mark_missing(nil, [1,2,3])).to be_empty
```

Comment:
Well. That »proper type casts« was indeed a bit flowery language. The only thing I wanted to do is check that this method will not blow up in my face. :laughing: The error is triggered by the `event_id's` I pass into the method. It will blow up mid-query. I don't care what this method returns. To test the fix a return value of `[]` is fine.

I suppose I could check for 
```ruby
expect(radar.mark_missing(nil, [1,2,3])).not_to raise_error
```

What do you think? :smiley_cat: